scripts/manage/push_config.sh

```diff
@@ -6,6 +6,8 @@ bold() {
 
 source ~/spinnaker-for-gcp/scripts/install/properties
 
+~/spinnaker-for-gcp/scripts/manage/check_project_mismatch.sh
```

Comment:
Calls to gcloud on lines 53, 58, and 61 were throwing a project empty string error if the user did not have a default project set.